lms/static/sass/views/_account-settings.scss

```diff
@@ -551,9 +551,10 @@
         margin: 20px 0;
       }
 
-      .paragon__modal-open {
+      .paragon__modal {
         overflow-y: scroll;
```

Comment:
Does this CSS override CSS imported from Paragon? I recently made a change to the Paragon modal so that the body will scroll if it overflows, so I'm not sure if this is needed.

Reply:
Thanks @thallada , this code was added for an earlier version of the modal so I will now remove it.

Reply:
I took a look at your sandbox: 
![screenshot-20180517105911-711x957](https://user-images.githubusercontent.com/1505923/40185763-6536bbbc-59c1-11e8-9c71-4e213e738deb.png)

I think that extra scrollbar is caused by this rule. Also, the margin rule here is causing the modal footer to fall off the bottom of the page a little.

Reply:
@thallada there used to be default margins around the modal, which are now gone. Do you know if this was a conscious design choice, or an unintended consequence of Turc's change?